src/vulkan-renderer/imgui/imgui.cpp

```diff
@@ -0,0 +1,616 @@
+#include "inexor/vulkan-renderer/imgui/imgui.hpp"
+#include "inexor/vulkan-renderer/wrapper/make_info.hpp"
+
+#include <cassert>
+#include <stdexcept>
+
+namespace inexor::vulkan_renderer::imgui {
+
+ImGUIOverlay::ImGUIOverlay(ImGUIOverlay &&other) noexcept
```

Comment:
Not sure if making `ImGUIOverlay` movable is worth it? I feel like it's not an object that would be moved.

Reply:
I will keep it for now.